.github/workflows/plugins-test.yaml

```diff
@@ -504,3 +504,28 @@ jobs:
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run mysql 5.1.2-8.0.15 (53)
         run: bash test/plugin/run.sh mysql-scenario
+
+  ShardingSphere3x:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 90
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build SkyWalking Agent
+        run: ./mvnw clean package -DskipTests -Pagent >/dev/null
+      - name: Build the Docker image
+        run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
+      - name: Run shardingsphere-3.x-scenario 3.0.0 (1)
+        run: bash test/plugin/run.sh shardingsphere-3.x-scenario
```

Comment:
I think we can add it to the fastest group. Because it has one version only.

Reply:
@tuohai666 more specifically, `Oracle_Kafka_JdkHttp_JdkThreading` is the fastest group now, moving to this group should be fine

Reply:
Thanks for your  comments. Can you tell me what is the fastest gropu? I can find this key word in doc, but not quite understand:

Please check the prev agent related PRs, and add your case to the fastest group, in order to make the whole test finished as soon as possible.

Reply:
Simply, check this

![image](https://user-images.githubusercontent.com/5441976/71255067-e17eab00-2367-11ea-9bd9-66682d6a5dfd.png)

Or more details at this page, https://github.com/apache/skywalking/pull/4093/checks?check_run_id=358026747

Reply:
> Thanks for your comments. Can you tell me what is the fastest gropu? I can find this key word in doc, but not quite understand:
> 
> Please check the prev agent related PRs, and add your case to the fastest group, in order to make the whole test finished as soon as possible.

You can find the time costs for each group in the check statuses in this page, see attached picture:

<img width="700" alt="image" src="https://user-images.githubusercontent.com/15965696/71255076-e5123200-2367-11ea-8941-a4128d92dfb5.png">

Reply:
Thanks for your quick response.